src/index.html

```diff
@@ -4,9 +4,321 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/blocks/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+     crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logoMOYO"
+          >
```

Comment:
```suggestion
        <img
          class="logo"
          src="./images/logo.png"
          alt="logoMOYO"
        >
```
Pay attention to formatting